cmd/generate-config/config/config-openapi-spec.json

```diff
@@ -267,6 +268,23 @@
           "type": "string"
         }
       }
+    },
+    "storage": {
+      "description": "Storage represents a subfield of the microshift config data structure. Its purpose to provide a user\nfacing interface to control whether MicroShift should deploy LVMS on startup.",
+      "type": "object",
+      "properties": {
+        "driver": {
+          "description": "DriverOpt is an enum value that determines whether MicroShift deploys LVMS.",
+          "type": "string"
```

Comment:
Should we specify allowed values here?

Reply:
I mean, is there something like enum in the specification to check for values?
```
    "enum": ["", "lvms", "none"]
```

Reply:
From https://book.kubebuilder.io/reference/markers/crd-validation:
> // +kubebuilder:validation:Enum

but not sure it'll handle unset value

Reply:
I'll test it. If it does, will add.

Reply:
Actually, it provides an "optional" check as well. Will combine both